src/gui/mesh/qgsmeshlayerproperties.h

```diff
@@ -47,6 +51,13 @@ class APP_EXPORT QgsMeshLayerProperties : public QgsOptionsDialogBase, private U
      */
     QgsMeshLayerProperties( QgsMapLayer *lyr, QgsMapCanvas *canvas, QWidget *parent = nullptr, Qt::WindowFlags = QgsGuiUtils::ModalDialogFlags );
 
+    /**
+     * Adds properties page from a factory
+     *
+     * \since QGIS 3.18
```

Comment:
@vcloarec 3.16 instead?

Reply:
First, I was thinking for 3.18, but as it is merged by Nyall, ok for 3.16!

Reply:
https://github.com/qgis/QGIS/pull/38734